packages/flutter/test/material/input_decorator_test.dart

```diff
@@ -4533,43 +4533,6 @@ void main() {
     expect(intrinsicHeight, equals(height));
   });
 
-  testWidgets('Error message for negative baseline', (WidgetTester tester) async {
```

Comment:
Why is this removed?

Reply:
I added that in https://github.com/flutter/flutter/issues/74326. There used to be a helper method that lays out a child and computes its baseline, even for center aligned children. It's removed. Having negative baselines may look ugly but I don't think it asserts anymore.